src/Tools/babylon.observable.ts

```diff
@@ -154,6 +154,36 @@
 
             return result;
         }
+
+        /**
+         * Does this observable has observers registered with pick mask
+         * @return {boolean} whether or not it has observers registered with pick mask
+        **/
+        public hasPickMasks(): boolean {

```

Comment:
This one can be removed. I would like to keep the observable as generic as possible. You can just use hasSpecificMask